src/main/scala/ch/epfl/bluebrain/nexus/iam/marshallers/instances.scala

```diff
@@ -129,6 +136,20 @@ object instances extends FailFastCirceSupport {
       }
     }
 
+  implicit val permissionsStatusCode: RejectionStatusCode[PermissionsRejection] =
+    new RejectionStatusCode[PermissionsRejection] {
+      override def apply(rej: PermissionsRejection): StatusCode = rej match {
```

Comment:
If all of them should yield a `BadRequest`, we can simplify.

Reply:
I think it's a good idea to keep them explicit such that when we add another one the compiler will complain...